taxonomies/ingredients.txt

```diff
@@ -62370,67 +62353,63 @@ en:lota lota
 la:lota lota
 
 
-# The capelin or caplin (Mallotus villosus) is a small forage fish of the smelt family found in the North Atlantic, North Pacific, and Arctic oceans.
+# en:description:The capelin or caplin (Mallotus villosus) is a small forage fish of the smelt family found in the North Atlantic, North Pacific, and Arctic oceans.
 
 <en:fish
-en:capelin, caplin
-ar:كبلين
-az:Moyva
-be:Мойва
-bg:мойва
-br:Kaplañsenn
-cs:huňáček severní
-cy:Caplan
-de:Lodde
-eu:Paneka atlantiko
-fa:خردماهی
-fi:Villakuore
-fr:capelan
-he:טרוטנית
-hu:Csuklyás hal
-id:Capelin
-is:Loðna
-it:Capelin
-ja:カラフトシシャモ
-jv:Capelin
-ka:მოივა
-kk:Мойва
-ko:열빙어
-lt:Stintenė
-mk:капелин
-ms:Capelin
-nb:lodde
-nl:lodde
-nn:Lodde
-pl:Gromadnik
-ru:Мойва
-so:Masago
-sv:Lodda
-uk:Мойва
-vi:Cá trứng
-zh:毛鱗魚
-# kk-arab:مويۆا
-# kk-cn:مويۆا
-# kk-cyrl:Мойва
-# kk-kz:Мойва
-# kk-latn:Moýva
-# kk-tr:Moýva
-# vep:Hohol'
-# zh-cn:毛鳞鱼
-# zh-hans:毛鳞鱼
-# zh-hant:毛鱗魚
-# zh-hk:毛鱗魚
-# zh-sg:毛鳞鱼
-# zh-tw:毛鱗魚
+en:capelin, caplin, Mallotus villosus
```

Comment:
For ingredients, we match latin automatically, there is no need to add it to all entries.